<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>                                                                                  Gestion Des Emissions</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>861</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(227, 227, 227);

font: 9pt &quot;Sylfaen&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>90</y>
        <width>20</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QTableView" name="table_AfficherEmission">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>741</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
     <widget class="QLabel" name="label_40">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);
image: url(:/new/prefix1/Logo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_44">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600;&quot;&gt;STREAMFLIX&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>881</width>
        <height>621</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_regarder">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRef">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/ac.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonTri">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>30</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
image: url(:/new/prefix1/Tri.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>verticalScrollBar</zorder>
     <zorder>table_AfficherEmission</zorder>
     <zorder>label_40</zorder>
     <zorder>label_44</zorder>
     <zorder>pushButton_regarder</zorder>
     <zorder>pushButtonRef</zorder>
     <zorder>label_14</zorder>
     <zorder>pushButtonTri</zorder>
     <zorder>label_20</zorder>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>Id Emission :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>181</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Réalisateur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>182</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Titre de l'Emission :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>221</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Durée :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>222</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>140</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Catégorie :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IdEmission">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Titre">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Categorie">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Realisateur">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Duree">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>220</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Description">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Ajouter">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>4</month>
        <day>7</day>
       </date>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_QuitterAjouter">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>851</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/Background.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>lineEdit_IdEmission</zorder>
     <zorder>lineEdit_Titre</zorder>
     <zorder>lineEdit_Categorie</zorder>
     <zorder>lineEdit_Realisateur</zorder>
     <zorder>lineEdit_Duree</zorder>
     <zorder>lineEdit_Description</zorder>
     <zorder>pushButton_Ajouter</zorder>
     <zorder>dateEdit</zorder>
     <zorder>pushButton_QuitterAjouter</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_QuitterModifier">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IdEmission_M">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Realisateur_M">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>222</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Durée :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>290</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description :</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_M">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>220</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="date">
       <date>
        <year>2024</year>
        <month>4</month>
        <day>7</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>181</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Titre de l'Emission :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Duree_M">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>220</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>222</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>140</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>Id Emission :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Titre_M">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Description_M">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>142</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Catégorie :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Categorie_M">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>140</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>180</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Réalisateur :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-30</y>
        <width>851</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/Background.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_recherID">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRechID">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_valider">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>390</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(155, 1, 1);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Id Emission :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/Recherche.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>pushButton_QuitterModifier</zorder>
     <zorder>lineEdit_IdEmission_M</zorder>
     <zorder>lineEdit_Realisateur_M</zorder>
     <zorder>label_9</zorder>
     <zorder>label_13</zorder>
     <zorder>dateEdit_M</zorder>
     <zorder>label_15</zorder>
     <zorder>lineEdit_Duree_M</zorder>
     <zorder>label_16</zorder>
     <zorder>label_17</zorder>
     <zorder>lineEdit_Titre_M</zorder>
     <zorder>lineEdit_Description_M</zorder>
     <zorder>label_18</zorder>
     <zorder>lineEdit_Categorie_M</zorder>
     <zorder>label_19</zorder>
     <zorder>lineEdit_recherID</zorder>
     <zorder>pushButtonRechID</zorder>
     <zorder>pushButton_valider</zorder>
     <zorder>label_21</zorder>
     <zorder>label_29</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>230</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(227, 227, 227);</string>
      </property>
      <property name="text">
       <string>Id Emission :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_RechercheIdSup">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>230</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_supprimer">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>230</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>851</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/Background.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonRecherche">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>270</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonsuptout">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>360</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);
border :1px solid rgb(255, 255, 255);
border-radius :10px;
color: #FFF;
padding-left: 20px;
padding-right:20px;
</string>
      </property>
      <property name="text">
       <string>Supprimer Tout</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>230</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/p.png);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>pushButtonRecherche</zorder>
     <zorder>label_4</zorder>
     <zorder>label_39</zorder>
     <zorder>lineEdit_RechercheIdSup</zorder>
     <zorder>pushButton_supprimer</zorder>
     <zorder>pushButtonsuptout</zorder>
     <zorder>label_22</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
